### ISSUE TITLE
Automatically 'approve' new ideas?

### ISSUE THREAD

**Issue by dep** [ivandiazwm/phpback#141]
I'd like for someone's idea to be immediately visible where others can vote on them without having to wait for an admin to Approve an idea to become visible. Is this possible?

**Comment by ChreativelyChris** [comment]
I would really like this feature as well.

**Comment by petrk94** [comment]
This were a great idea, looking for too

**Comment by petrk94** [comment]
This were a great idea, looking for too

**Comment by petrk94** [comment]
This were a great idea, looking for too

**Comment by drveresh** [comment]
From a security point of view, it is not safe.  A hacker may apply DoS, Replay-Attack or just automate posting new ideas and eventually spam the dashboard.  Sometimes, there could be duplicate or similar ideas, so Admin is the right person to evaluate and Approve it.

**Comment by petrk94** [comment]
@drveresh what about solving this issue another way?
The account what create get 3 statuses (signed up, but locked; unverified; verified account)

- signed up, but locked(user need email opt-in approval for usage) 
- unverified (accounts who get approved by email recipient) (admin has to approve account after email opt-in)
- verified account (account which has proved the existence of human behind, verification process through admin or Captcha questions and providing information like phone number or other identity keys, like Twitter or Facebook profile)

Then, only verified user postings get approved automatically, not verified user get approved by admin (admin get email and it contains link for approval)

Thats issue is solvable.

**Comment by petrk94** [comment]
Do you have feedback to my idea how to solve this issue?

**Comment by drveresh** [comment]
@petrk94 I agree with you and may need to add additional states to the user profile to achieve this feature.  Also, currently the email service doesn't seem to be working, so let me check this first and then scope this auto-approve feature.
